### DISCUSSION TITLE
Software-Maya（建模篇） | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#326] [Announcements]
# Software-Maya（建模篇） | Zi-Zi's Journey

学习自中国大学 MOOC——Maya 三维动画制作案例教程-郑州轻工业大学。

https://gz-metal-cell.github.io/2024/04/01/Software-Maya%EF%BC%88%E5%BB%BA%E6%A8%A1%E7%AF%87%EF%BC%89/

**Comment by promefire** [comment]
😭正所谓艺术来源于生活😭Metal哥得加强一下python进阶学习😭要不然做出来的胸部就像个没有顶的[金字塔](https://gz-metal-cell.github.io/2024/04/01/Software-Maya%EF%BC%88%E5%BB%BA%E6%A8%A1%E7%AF%87%EF%BC%89/4.3.45.png)😭

**Reply by GZ-Metal-Cell** [reply]
😭没学过人体素描😭太难捏了😭学 python 还是得向凡哥学习😭
